### PR TITLE
Cypress test. Reset password notification.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Cypress test for #3140 
Increase coverage.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by dvkruchinin** [comment]
Added a dummy email. So as when specifying the email with which the user is registered an error
```
[Thu Apr 29 06:18:11.449787 2021] [wsgi:error] [pid 401:tid 140636014475008] [remote 172.28.0.6:34008]   File "/opt/venv/lib/python3.8/site-packages/django/core/mail/backends/smtp.py", line 102, 
in send_messages
[Thu Apr 29 06:18:11.449790 2021] [wsgi:error] [pid 401:tid 140636014475008] [remote 172.28.0.6:34008]     new_conn_created = self.open()
[Thu Apr 29 06:18:11.449793 2021] [wsgi:error] [pid 401:tid 140636014475008] [remote 172.28.0.6:34008]   File "/opt/venv/lib/python3.8/site-packages/django/core/mail/backends/smtp.py", line 62, in open
[Thu Apr 29 06:18:11.449797 2021] [wsgi:error] [pid 401:tid 140636014475008] [remote 172.28.0.6:34008]     self.connection = self.connection_class(self.host, self.port, **connection_params)      
[Thu Apr 29 06:18:11.449801 2021] [wsgi:error] [pid 401:tid 140636014475008] [remote 172.28.0.6:34008]   File "/usr/lib/python3.8/smtplib.py", line 253, in __init__
[Thu Apr 29 06:18:11.449805 2021] [wsgi:error] [pid 401:tid 140636014475008] [remote 172.28.0.6:34008]     (code, msg) = self.connect(host, port)
[Thu Apr 29 06:18:11.449808 2021] [wsgi:error] [pid 401:tid 140636014475008] [remote 172.28.0.6:34008]   File "/usr/lib/python3.8/smtplib.py", line 339, in connect
[Thu Apr 29 06:18:11.449811 2021] [wsgi:error] [pid 401:tid 140636014475008] [remote 172.28.0.6:34008]     self.sock = self._get_socket(host, port, self.timeout)
```
 is issued. Perhaps this is the correct behavior since there is no smtp server.

File: tests/cypress/integration/actions_users/case_73_reset_password_notification.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Reset password notification.', () => {
+    const caseId = '73';
+    const dummyEmail = 'admin@local.local'
```
Comment (ActiveChooN) [inline tests/cypress/integration/actions_users/case_73_reset_password_notification.js:9]: Why not use email from `cypress.json`?

Comment (dvkruchinin) [reply tests/cypress/integration/actions_users/case_73_reset_password_notification.js:9]: Please take a look for my comment above. I got an error.

Comment (ActiveChooN) [reply tests/cypress/integration/actions_users/case_73_reset_password_notification.js:9]: Sure, didn't notice your comment.

**Review by ActiveChooN — APPROVED** [review]

**Pull request merged**